Example/Account/UserAccountView/UserAccountView.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import SwiftUI
+import Combine
+import CombineFeedback
+import CombineFeedbackUI
+
+public struct UserAccountView: View {
+
+    public typealias State = UserAccountViewModel.State
+    public typealias Event = UserAccountViewModel.Event
+
+    private let context: Context<State, Event>
+
+    private let loginViewModel: LoginViewModel
+    private let registrationViewModel: RegistrationViewModel
+
+    private let authenticationSubject: PassthroughSubject<LoggedAccount?, Never> = PassthroughSubject()
+
+    private let cancellable: AnyCancellable
+
+    public init(context: Context<State, Event>) {
+        self.context = context
+        self.loginViewModel = LoginViewModel()
```

Comment:
I'm not happy about this 😞

Reply:
But I don't really know who is responsible for instantiating view models, as it might need to receive some dependencies and would be nice to be able to mock them when testing.